.github/workflows/build.yml

```diff
@@ -4,19 +4,19 @@ on:
   push:
     branches:
       - master
+      - REL_16_STABLE
       - REL_15_STABLE
       - REL_14_STABLE
       - REL_13_STABLE
       - REL_12_STABLE
-      - REL_11_STABLE
   pull_request:
     branches:
       - master
+      - REL_16_STABLE
       - REL_15_STABLE
       - REL_14_STABLE
       - REL_13_STABLE
       - REL_12_STABLE
-      - REL_11_STABLE
```

Comment:
Will you release pg_rman for v11 in Janually?
I think it's better to remove the line after that.

Additionally you need remove line:49.

Reply:
Understood. I will remove after pg16 support.